src/object.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @author acvetkov@yandex-team.ru
+ * @overview
+ */
+
+'use strict';
+
+/**
+ * Get nested object by path
+ * @param {Object} object
+ * @param {String} path
+ * @param {String} [delimiter]
+ * @returns {Object|Null}
+ */
+export function get(object, path, delimiter = '.') {
+    var [...keys] = path.split(delimiter);
+    var data = object;
+
+    for (var i = 0, max = keys.length; i < max; i++) {
```

Comment:
You shouldn't cache length. It will not make a loop faster: http://mrale.ph/blog/2014/12/24/array-length-caching.html